### ISSUE TITLE
NetNewsWire missing 'author' tags in feed

### ISSUE THREAD

**Issue by dumprop** [FreshRSS/FreshRSS#3518]
**Describe the bug**
Subject

**To Reproduce**
1. Go to freshrss
2. Click on add feed
3. Enter https://feed.eugenemolotov.ru/?action=display&bridge=Twitter&context=By+username&u=durov&format=Atom
4. See the difference

**Expected behavior**
![image](https://user-images.githubusercontent.com/34004367/111068921-95ff7200-84c2-11eb-97c8-94e814e77f87.png)

**Screenshots**
![image](https://user-images.githubusercontent.com/34004367/111068964-c810d400-84c2-11eb-8563-754c2fba7fb5.png)

**Environment information (please complete the following information):**
 - FreshRSS version: 12 March 2021 at 14:18

**Comment by Alkarex** [comment]
Hello,
I cannot reproduce the problem, and I am not sure to understand the screenshots. Are they from an app?

I get the author field properly:
![image](https://user-images.githubusercontent.com/1008324/111074927-b12fa900-84e5-11eb-93d1-49a57a074e8c.png)

**Comment by dumprop** [comment]
> Hello,
> I cannot reproduce the problem, and I am not sure to understand the screenshots. Are they from an app?
> 
> I get the author field properly:
> ![image](https://user-images.githubusercontent.com/1008324/111074927-b12fa900-84e5-11eb-93d1-49a57a074e8c.png)

I have it in FreshRSS too
<img width="1130" alt="image" src="https://user-images.githubusercontent.com/34004367/111166808-66bb3480-8598-11eb-99db-480e09a1b58a.png">

But I dont have it in NetNewsWire
<img width="1130" alt="image" src="https://user-images.githubusercontent.com/34004367/111166843-733f8d00-8598-11eb-8f5f-6801143ba73c.png">
<img width="469" alt="image" src="https://user-images.githubusercontent.com/34004367/111167014-98cc9680-8598-11eb-8b12-081e5021f4c8.png">
 Is a NNW problem or FreshRSS api?

**Comment by dumprop** [comment]
<img width="675" alt="image" src="https://user-images.githubusercontent.com/34004367/111167790-5bb4d400-8599-11eb-8fae-92d1eb019b57.png">
 Fluent reader have author tag, so seems problem in NNW & FreshRSS API

**Comment by Alkarex** [comment]
FreshRSS supports two APIs: the Google Reader API (best), and Fever API (which is more limited and less efficient).

NetNewsWire uses the Fever API.

Both APIs contain the author information, as visible from the responses:

FreshRSS Google Reader API:

```json
{
	"id": "user/-/state/com.google/reading-list",
	"updated": 1615818199,
	"items": [{
			"id": "tag:google.com,2005:reader/item/0005bd93eb4cbef5",
			"crawlTimeMsec": "1615817889070",
			"timestampUsec": "1615817889070837",
			"published": 1610999100,
			"title": "Public calls to violence have no place on Telegram t.me/durov/149",
			"summary": {
				"content": "<div>\n\t<a href=\"https://twitter.com/durov\">\n<img alt=\"durov\" src=\"https://pbs.twimg.com/profile_images/658376777258151936/-Jz8l4Rr_normal.jpg\" title=\"Pavel Durov\"></a>\n</div>\n<div>\n\t<blockquote>Public calls to violence have no place on Telegram <a href=\"https://t.me/durov/149\">t.me/durov/149</a></blockquote>\n</div>\n<div>\n\t<blockquote></blockquote>\n</div>"
			},
			"canonical": [{
					"href": "https://twitter.com/durov/status/1351254307094618113"
				}
			],
			"alternate": [{
					"href": "https://twitter.com/durov/status/1351254307094618113"
				}
			],
			"categories": ["user/-/state/com.google/reading-list", "user/-/label/Uncategorized"],
			"origin": {
				"streamId": "feed/237",
				"title": "Twitter @durov"
			},
			"author": "Pavel Durov (@durov)"
		}, {
			"id": "tag:google.com,2005:reader/item/0005bd93eb4cbef4",
			"crawlTimeMsec": "1615817889070",
			"timestampUsec": "1615817889070836",
			"published": 1608736500,
			"title": "Today I outlined the monetization strategy of Telegram. It will allow us to remain independent and stay true to our values for decades to come – t.me/durov/142",
			"summary": {
				"content": "<div>\n\t<a href=\"https://twitter.com/durov\">\n<img alt=\"durov\" src=\"https://pbs.twimg.com/profile_images/658376777258151936/-Jz8l4Rr_normal.jpg\" title=\"Pavel Durov\"></a>\n</div>\n<div>\n\t<blockquote>Today I outlined the monetization strategy of Telegram. It will allow us to remain independent and stay true to our values for decades to come – <a href=\"https://t.me/durov/142\">t.me/durov/142</a></blockquote>\n</div>\n<div>\n\t<blockquote></blockquote>\n</div>"
			},
			"canonical": [{
					"href": "https://twitter.com/durov/status/1341764320510808067"
				}
			],
			"alternate": [{
					"href": "https://twitter.com/durov/status/1341764320510808067"
				}
			],
			"categories": ["user/-/state/com.google/reading-list", "user/-/label/Uncategorized", "user/-/state/com.google/read"],
			"origin": {
				"streamId": "feed/237",
				"title": "Twitter @durov"
			},
			"author": "Pavel Durov (@durov)"
		}
	],
	"continuation": "1615817889070836"
}
```

FreshRSS Fever API:

```json
{
	"api_version": 3,
	"auth": 1,
	"last_refreshed_on_time": 1615624441,
	"total_items": 239902,
	"items": [
		{
			"id": "1615817889070837",
			"feed_id": 237,
			"title": "Public calls to violence have no place on Telegram t.me\/durov\/149",
			"author": "Pavel Durov (@durov)",
			"html": "<div>\n\t<a href=\"https:\/\/twitter.com\/durov\">\n<img alt=\"durov\" src=\"https:\/\/pbs.twimg.com\/profile_images\/658376777258151936\/-Jz8l4Rr_normal.jpg\" title=\"Pavel Durov\"><\/a>\n<\/div>\n<div>\n\t<blockquote>Public calls to violence have no place on Telegram <a href=\"https:\/\/t.me\/durov\/149\">t.me\/durov\/149<\/a><\/blockquote>\n<\/div>\n<div>\n\t<blockquote><\/blockquote>\n<\/div>",
			"url": "https:\/\/twitter.com\/durov\/status\/1351254307094618113",
			"is_saved": 0,
			"is_read": 0,
			"created_on_time": 1610999100
		}
	]
}
```

As far as I can see, all is good on FreshRSS side.

**Comment by dumprop** [comment]
Yes, fluent reader contains author info

https://github.com/Ranchero-Software/NetNewsWire/issues/2890
